packages/manager/apps/web/client/app/domain/zone/product-zone-activate/activate.routing.js

```diff
@@ -0,0 +1,107 @@
+import get from 'lodash/get';
+
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('app.domain.product.zone-activate', {
```

Comment:
What was the issue with keeping 
```suggestion
  $stateProvider.state('app.domain.product.zone.activate', {
```
? 
That way we inherit from url (no need to prefix with `/zone`) but also from resolves that might be brought to `zone` (I don't know if they are of any use for activation)